pages/about-style.css

```diff
@@ -0,0 +1,126 @@
+
+.container {
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr;
+    grid-template-columns: 1fr 1fr 1fr;
+    background-color: rgb(252, 246, 229);
+    height: 100vh;
+    width:100vw;
+    position: relative;
+    /* border: solid red; */
+}
+
+.quote {
+    grid-column: 1 /2;
+    grid-row: 1 / 4;
+    display: grid;
+    align-items: center;
+    /* position: absolute; */
+    
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    font-size: 75px;
+    line-height: 65px;
```

Comment:
In the future consider using relative sizing so it can resize for smaller windows.